bin/Debugger.Sample/ChakraCore.Debugger.Sample.vcxproj

```diff
@@ -90,6 +98,7 @@
     <Import Project="..\..\packages\boost.1.68.0.0\build\boost.targets" Condition="Exists('..\..\packages\boost.1.68.0.0\build\boost.targets')" />
     <Import Project="..\..\packages\boost_date_time-vc141.1.68.0.0\build\boost_date_time-vc141.targets" Condition="Exists('..\..\packages\boost_date_time-vc141.1.68.0.0\build\boost_date_time-vc141.targets')" />
     <Import Project="..\..\packages\Microsoft.ChakraCore.vc140.1.11.20\build\native\Microsoft.ChakraCore.vc140.targets" Condition="Exists('..\..\packages\Microsoft.ChakraCore.vc140.1.11.20\build\native\Microsoft.ChakraCore.vc140.targets')" />
+    <Import Project="..\..\packages\ReactWindows.ChakraCore.ARM64.1.11.20\build\native\ReactWindows.ChakraCore.ARM64.targets" Condition="'$(Platform)' == 'ARM64' AND Exists('..\..\packages\ReactWindows.ChakraCore.ARM64.1.11.20\build\native\ReactWindows.ChakraCore.ARM64.targets')" />
```

Comment:
Could you explain what's required to be imported for ARM64? These should not be coming from the ReactNative project.

Reply:
Because the ChakraCore team is not producing an ARM64-enabled NuGet. They don't plan to do so.
They did provide us with the actual binaries. We just had to package them ourselves.

Reply:
Let's have a discussion about this. If they are providing the binaries, then they will need to do so for each release. Why wouldn't ChakraCore then be creating the NuGet? Could we have the ChakraCore-Debugger pipeline create the separate NuGet for ARM64 rather than do this from ReactWindows?

Reply:
> Why wouldn't ChakraCore then be creating the NuGet?
Can't say in detail. We'd have to ask them. They did say that, because ChakraCore is in "life-support" mode, they are reluctant to modify their release pipeline just to provide support for a new platform.

> Could we have the ChakraCore-Debugger pipeline create the separate NuGet for ARM64 rather than do this from ReactWindows?
Is that:
- A separate ChakraCore|ARM64 NuGet?
  It's already produced and released. Yes, updated binaries will need to be handed by them, but to have the CCDB pipeline produce updated ARM64 NuGets, they would already need to place them in a storage source accessible by the pipeline. What are we gaining out of this, and how does that block this PR?
- A separate ChakraCore-Debugger|ARM64 NuGet?
  What is the use for this? We currently bundle both x86 and x64 binaries in the same NuGet. We would then have to create separate NuGets for x86 and x64, shouldn't we?

Reply:
A separate ChakraCore ARM64 NuGet. Regardless of who produced this NuGet, it doesn't have anything to do with ReactWindows other than it wants to be a consumer of the NuGet. Couldn't the name be updated to reflect that by using "Microsoft" instead of "ReactWindows"?

Reply:
`Microsoft` is considered a reserved prefix, which should be only used for officially-released packages.
The "ReactWindows" prefix is used for many semi-internal (internally produced, but publicly available) packages.

Deciding on the naming convention of other projects' NuGet packages is out of the scope of this PR.
There is no need nor use for a new/renamed package, especially when the existing one is already also used by React Native Windows.

If the PR itself does not introduce any regression, it doesn't add any build or runtime dependencies on React Native, and does not affect any functional or conventional content of the code base, can it be merged as is?